src/Analyser/LatteContextData.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+declare (strict_types=1);
+
+namespace Efabrica\PHPStanLatte\Analyser;
+
+use Efabrica\PHPStanLatte\Collector\ValueObject\CollectedValueObject;
+use PHPStan\Analyser\Error;
+
+class LatteContextData
+{
+    /**
+     * @var array<Error>
+     */
+    private $errors;
+
+    /**
+     * @var array<CollectedValueObject>
+     */
+    private $collectedData = [];
+
+    /**
+     * @var array<class-string, CollectedValueObject[]>
+     */
+    private $collectedDataByType = [];
+
+    /**
+     * @param array<Error> $errors
+     * @param array<CollectedValueObject> $collectedData
+     */
+    public function __construct(array $errors, array $collectedData)
```

Comment:
Why errors is first argument, aren't collected data more important?

Reply:
I copied it for FileAnalyserResult and did not changed it. But agreed tht in our use case collected data is more important and should be firt parameter.